airbyte-integrations/connectors/source-faker/unit_tests/unit_test.py

```diff
@@ -0,0 +1,91 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+# import jsonschema
+from airbyte_cdk.models import AirbyteMessage, Type
+from source_faker import SourceFaker
+
+
+# TODO: airbyte/airbyte-integrations/connectors/source-faker/.venv/lib/python3.9/site-packages/airbyte_cdk/sources/utils/transform.py:12: DeprecationWarning: Call to deprecated class AirbyteLogger. (Use logging.getLogger('airbyte') instead) -- Deprecated since version 0.1.47.
```

Comment:
```
airbyte/airbyte-integrations/connectors/source-faker/.venv/lib/python3.9/site-packages/airbyte_cdk/sources/utils/transform.py:12: DeprecationWarning: Call to deprecated class AirbyteLogger. (Use logging.getLogger('airbyte') instead) -- Deprecated since version 0.1.47.
```
How do I fix this warning?